packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -32,6 +33,13 @@ const newController = async (options: any) => {
   await networkController.initializeProvider();
   const { provider } = networkController.getProviderAndBlockTracker();
 
+  new ApprovalController({
```

Comment:
we may need to expose this since we will probably have to call some method on this controller to approve a request